app/src/main/java/com/spendster/presentation/mainScreen/ActivityActivity.java

```diff
@@ -0,0 +1,126 @@
+package com.spendster.presentation.mainScreen;
+
+import android.graphics.Color;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.spendster.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ActivityActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private List<Fragment> pageFragments;
+    private ViewPager viewPager;
+    private Button btnAdd, btnActivity, btnOverview, btnBudget, btnProfile;
```

Comment:
@SimonMaha in order to make a single layout, you can use not buttons with click listeners on them, but root layout.